<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="NoEqn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter number of Equations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterNo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Eqn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Coefficients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterEqn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QComboBox" name="method">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Gauss Elimination</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss Jordan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LU decompostion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jacobi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss sidel</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="FindSol">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Solution</string>
    </property>
   </widget>
   <widget class="QLabel" name="methodLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="No">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLabel" name="var">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ParametersLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Parameters">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Doolittle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Crout</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cholesky</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="SigFigures">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Number of Siginificant Figures (default : 5)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterSigFigures">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>40</y>
      <width>20</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="StoppingCondition">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="StoppingConditionLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stopping Condition</string>
    </property>
   </widget>
   <widget class="QLabel" name="IterationLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IterationNumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InitiaGuessLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial guess</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitialGuess">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Default : zeros, Enter:0,0,0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Solution</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>219</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="result">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>221</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Time spent</string>
    </property>
    <widget class="QLabel" name="time">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Iterations Number</string>
    </property>
    <widget class="QLabel" name="Iterations">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Equations</string>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>219</width>
        <height>219</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="view">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>221</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
